content/installation/python/PythonConfig.md

```diff
@@ -0,0 +1,46 @@
+<!-- 
+title: "Python Agent Configuration"
+description: "Configuring the Python Agent and Service"
+tags: "installation python django flask pyramid agent service configuration"
+-->
+
+The Python agent and Go service use a YAML file to alter the agent behavior. 
+
+## Load Path
+
+The configuration file is called *contrast_security.yaml* wherever it's located. The Python agent loads the configuration YAML from the following paths in order of precedence:
+
+1. The current working directory (e.g., *./contrast_security.yaml*)
+2. A subdirectory called *config*, (e.g., *./config/contrast_security.yaml*)
+3. Within the server's *etc* directory (e.g., */etc/contrast_security.yaml*)
+
+
+## General Configuration Options
+
+The configuration YAML consists of four sections. The agent and service may share a common configuration file, but only some options and sections are applicable to each process.
+
+* `teamserver`: Options for locating and communicating with the Contrast interface Dashboard
```

Comment:
This is incorrect. This structure should be based on the common config style.  Looks like I need to update this for Ruby, too.

Reply:
Yes I think we need a common config page. I can remove this and make a new ticket for that.

Is that good?

Reply:
I just submitted a PR to update for Ruby. Match that?

Reply:
Yep! Thanks!

Reply:
Done!